mmv1/products/notebooks/api.yaml

```diff
@@ -462,6 +466,480 @@ objects:
+                name: "encryptionConfig"
+                description: |
+                  Encryption settings for virtual machine data disk.
+                properties:
+                  - !ruby/object:Api::Type::String
+                    name: "kmsKey"
+                    description: |
+                      The Cloud KMS resource identifier of the customer-managed
+                      encryption key used to protect a resource, such as a disks.
+                      It has the following format:
+                      `projects/{PROJECT_ID}/locations/{REGION}/keyRings/
+                      {KEY_RING_NAME}/cryptoKeys/{KEY_NAME}`
+              - !ruby/object:Api::Type::NestedObject
+                name: "shieldedInstanceConfig"
+                description: |
+                  Shielded VM Instance configuration settings.
+                properties:
+                  - !ruby/object:Api::Type::Boolean
+                    name: "enableSecureBoot"
+                    description: |
+                      Defines whether the instance has Secure Boot enabled.Secure
+                      Boot helps ensure that the system only runs authentic software
+                      by verifying the digital signature of all boot components, and
+                      halting the boot process if signature verification fails.
+                      Disabled by default.
+                  - !ruby/object:Api::Type::Boolean
+                    name: "enableVtpm"
+                    description: |
+                      Defines whether the instance has the vTPM enabled. Enabled by
+                      default.
+                  - !ruby/object:Api::Type::Boolean
+                    name: "enableIntegrityMonitoring"
+                    description: |
+                      Defines whether the instance has integrity monitoring enabled.
+                      Enables monitoring and attestation of the boot integrity of
+                      the instance. The attestation is performed against the
+                      integrity policy baseline. This baseline is initially derived
+                      from the implicitly trusted boot image when the instance is
+                      created. Enabled by default.
+              - !ruby/object:Api::Type::NestedObject
+                name: "acceleratorConfig"
+                description: |
+                  The Compute Engine accelerator configuration for this runtime.
+                properties:
+                  - !ruby/object:Api::Type::Enum
+                    name: "type"
+                    description: |
+                      Accelerator model.
+                    values:
+                      - ACCELERATOR_TYPE_UNSPECIFIED
+                      - NVIDIA_TESLA_K80
+                      - NVIDIA_TESLA_P100
+                      - NVIDIA_TESLA_V100
+                      - NVIDIA_TESLA_P4
+                      - NVIDIA_TESLA_T4
+                      - NVIDIA_TESLA_A100
```

Comment:
same here, we should probably maintain this as a string with reference to the page where these types are defined

https://cloud.google.com/vertex-ai/docs/workbench/reference/rest/v1/projects.locations.runtimes#AcceleratorType

Reply:
Done